addons/Yush/Yush.lua

```diff
@@ -127,7 +141,9 @@ parse_binds = function(fbinds, top)
 
     rawset(names, top, rawget(_innerG._names, fbinds))
     for key, val in pairs(fbinds) do
+        key_combo = key
```

Comment:
Either make this variable `local` or even better, change line 145 so it doesn't reassign `key`, but instead using a new variable. This was weird before, so not your fault. But since you're here already, you may as well do me a favor and adjust this :)